tests/sentry/sentry_apps/tasks/test_sentry_apps.py

```diff
@@ -625,6 +628,136 @@ def test_error_created_sends_webhook(self, mock_record, safe_urlopen):
             mock_record=mock_record, outcome=EventLifecycleOutcome.SUCCESS, outcome_count=4
         )
 
+    @responses.activate
+    @patch("sentry.integrations.utils.metrics.EventLifecycle.record_event")
+    def test_project_filter_no_filters_sends_webhook(self, mock_record, safe_urlopen):
+        create_or_update_service_hooks_for_installation(
+            installation=self.install,
+            webhook_url=self.sentry_app.webhook_url,
+            events=self.sentry_app.events,
+        )
+
+        event = self.store_event(data={}, project_id=self.project.id)
+        assert event.group is not None
+        with self.tasks():
+            post_process_group(
+                is_new=True,
+                is_regression=False,
+                is_new_group_environment=False,
+                cache_key=write_event_to_cache(event),
+                group_id=event.group_id,
+                project_id=self.project.id,
+                eventstream_type=EventStreamEventType.Error,
+            )
+
+        assert safe_urlopen.called
+        ((args, kwargs),) = safe_urlopen.call_args_list
+        data = json.loads(kwargs["data"])
+        assert data["action"] == "created"
+        assert data["installation"]["uuid"] == self.install.uuid
+        assert data["data"]["issue"]["id"] == str(event.group.id)
+
+        # SLO assertions
+        assert_success_metric(mock_record)
+        # PREPARE_WEBHOOK (success) -> SEND_WEBHOOK (success) x 1
+        assert_count_of_metric(
+            mock_record=mock_record, outcome=EventLifecycleOutcome.STARTED, outcome_count=4
```

Comment:
@Christinarlong I added these like you requested but not sure I fully understand how these event lifecycle outcomes work, so just put whatever was needed for assertions to succeed. 

Could you please check these values (e.g. `outcome_count=4`) make sense in this and the other two test?

Reply:
added suggestions on and see comment below for an explanation

Reply:
thanks!